pages/linux/lxterminal.md

```diff
@@ -0,0 +1,24 @@
+# lxterminal
+
+> Terminal emulator for LXDE.
+> More information: <https://wiki.lxde.org/en/LXTerminal>.
+
+- Open an lxterminal window:
+
+`lxterminal`
+
+- Open an lxterminal window, run a command and exit:
+
+`lxterminal -e "{{command}}"`
+
+- Open an lxterminal window with multiple tabs:
+
+`lxterminal --tabs={{tab_name1,tab_name2...}}`
+
+- Open an lxterminal window with a specific title:
+
+`lxterminal --title={{title_name}}`
+
+- Open an lxterminal window with a specific working directory:
+
+`lxterminal --working-directory={{/path/to/directory}}`
```

Comment:
Relative paths are not possible?

Reply:
It take a working directory, not a file

its just cd'ing into it

Reply:
@marchersimon is asking if this valid (I tested and it's valid)
```suggestion
`lxterminal --working-directory={{path/to/directory}}`
```

Reply:
Where does it start from?

Reply:
from the current directory, e.g. you can use:

```bash
lxterminal --working-directory=..
```

Reply:
.. from where

/home? from ~

Reply:
@CleanMachine1 from the current directory, yeah you can Use whatever you want:
`--working-directory=~/Downloads`, `--working-directory=../another_folder`, `--working-directory=/root`, etc.